src/config/services/index.md

```diff
@@ -32,3 +37,136 @@ Optionally, a service directory may contain:
 
 When a new service is created, a `supervise` folder will be automatically
 created on the first run.
+
+### Configuring Services
+
+Most services can take configuration options set by a `conf` file in the service
+directory. This allows service customization without modifying the service
+directory provided by the relevant package.
+
+Check the service file for how to pass configuration parameters. A few services
+have a field like `OPTS="--value ..."` in their `conf` file.
+
+To make more complex customizations, you should [edit the
+service](#editing-services).
+
+### Editing Services
+
+To edit a service, first copy its service directory to a different directory
+name. Otherwise, [xbps-install(1)](https://man.voidlinux.org/xbps-install.1) can
+overwrite the service directory. Then, edit the new service file as needed.
+Finally, the old service should be stopped and disabled, and the new one should
+be started.
+
+## Managing Services
+
+### Runsvdirs
+
+A `runsvdir` is a directory in `/etc/runit/runsvdir` containing enabled services
+in the form of symlinks to service directories. The currently running `runsvdir`
+will be linked to `/var/service` when the system is booted.
+
+The `runit-void` package comes with two runsvdir directories; `single` and
+`default`.
+
+- `single` just runs sulogin(8) and the necessary steps to rescue your system.
+- `default` the default runsvdir.
+
+Additional runsvdirs can be created in `/etc/runit/runsvdir/`.
+
+See [runsvdir(8)](https://man.voidlinux.org/runsvdir.8) and
+[runsvchdir(8)](https://man.voidlinux.org/runsvchdir.8) for further information.
+
+#### Booting a different runsvdir
+
++To boot a runsvdir other than `default`, the name of the desired runsvdir +can
+be added to the [kernel command-line](../kernel.html#cmdline). As an +example,
+adding `single` to the kernel command line will boot the `single` +runsvdir.
```

Comment:
Extraneous "+"s in this para? Also, change the link to use "kernel.md".

Reply:
Yeah, copying from a commit diff has its disadvantages. Thanks!